webviz_subsurface/datainput/__init__.py

```diff
@@ -6,8 +6,10 @@
 
 from ._history_match import extract_mismatch, scratch_ensemble
 from ._intersect import load_surface, get_wfence, get_hfence
-from ._summary_stats import get_summary_data, get_summary_stats
 from ._inplace_volumes import extract_volumes
+from ._reservoirsimulationrimeseries import get_time_series_data, \
```

Comment:
Misspelling. Should be e.g. `._reservoirsimulationrimeseries` instead of `._reservoirsimulationtimeseries`.